.github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md

```diff
@@ -0,0 +1,54 @@
+---
+name:  Dependencies Issue
+about: Use this template for all Dependencies Issues, even those automatically generated
+labels: dependencies
+---
+
+# Description
+- Information about the dependency and its role in the project
+- Current version - what are we running
+- Target Version - what are we changing it to
+- DependaBot PR link if available
+
+## Features and fixes available
+- List of important bug fixes, features or security patches
```

Comment:
Are we sure this information isn't redundent? it's all generated along side the PR with commits etc.

Reply:
I would think that at the Issue level, we have the generic list of things in the updated version and in the PR we would have specifics about what we leveraged, if anything. In some cases we just verify that the existing code works with the new version, correct?

Reply:
yeah, that is usually the case from what I've seen. Most will just come through as a PR that once approved by reviewers, if tests are passing, gets merged.

Reply:
ok then no changes necessary. Issue has general list, PR has specifics.